tools/migrations/2021_12_15_addSubmittedAtField.js

```diff
@@ -0,0 +1,9 @@
+export const up = async ({ sequelize, Sequelize }) => {
```

Comment:
This is great and very thoughtful but I don't think we'll actually need to run this. Sequelize _should_ create new columns for us when we add them to our models (and fwiw, since this model isn't in production yet, we can always just wipe out the table on dev instead of performing a migration of any complexity if it saves us trouble)